From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Fri, 9 Mar 2018 22:04:42 +0100
Subject: btmp config: allow rw group access

---
 examples/btmp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/btmp b/examples/btmp
index 393ead5..0aa1ae1 100644
--- a/examples/btmp
+++ b/examples/btmp
@@ -2,6 +2,6 @@
 /var/log/btmp {
     missingok
     monthly
-    create 0600 root utmp
+    create 0660 root utmp
     rotate 1
 }
