Index: logrotate.git/logrotate.c
===================================================================
--- logrotate.git.orig/logrotate.c	2016-12-29 02:36:21.851158797 +0100
+++ logrotate.git/logrotate.c	2016-12-29 02:36:21.848158800 +0100
@@ -496,8 +496,12 @@
     if (fchmod(fd, (S_IRUSR | S_IWUSR) & sb->st_mode)) {
 	message(MESS_ERROR, "error setting mode of %s: %s\n",
 		fileName, strerror(errno));
-	close(fd);
-	return -1;
+	if (geteuid() == 0) {
+		/* If run as non-root, we may not be able to duplicate
+		   the ownership of the file. Warn but don't fail. */
+		close(fd);
+		return -1;
+	}
     }
 
 	if (fstat(fd, &sb_create)) {
